dts/arm/silabs/siwg917.dtsi

```diff
@@ -125,13 +125,22 @@
 			status = "disabled";
 		};
 
+		siwx91x_soc_pd: siwx91x-soc-pd {
+			compatible = "power-domain";
```

Comment:
Since the power domain needs a dedicated driver, shouldn't there be a specific binding for SiWx91x power domain?